tests/ops/op_math/test_adjoint.py

```diff
@@ -884,6 +884,16 @@ def test_single_observable(self):
         qs = qml.tape.QuantumScript.from_queue(q)
         assert len(qs) == 0
 
+    def test_correct_queued_operators(self):
+        """Test that args and kwargs do not add operators to the queue."""
+
+        with qml.queuing.AnnotatedQueue() as q:
+            qml.adjoint(qml.QSVT)(qml.X(1), [qml.Z(1)])
+            qml.adjoint(qml.QSVT(qml.X(1), [qml.Z(1)]))
+
+        for op in q.queue:
+            assert op.name == "Adjoint(QSVT)"
```

Comment:
In the test, can you also check that the length of the queue is 2?

Reply:
Sure :)